java/RSPEC-1994.java

```diff
@@ -0,0 +1,10 @@
+class ForLoopVarModify {
```

Comment:
RSPEC-2761: Don't repeat Unary operators
<!-- lookout comment separator -->
---
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._